pkg/hyper/helper.go

```diff
@@ -358,3 +357,26 @@ func promiseGo(f func() error) chan error {
 	}()
 	return ch
 }
+
+// checkIfContainerRunning check if container is running by containerID
+func checkIfContainerRunning(client *Client, containerID string) error {
```

Comment:
this function is better to be renamed to `ensureContainerRunning` since it returns error instead of bool